doc/proxy-clients.rst

```diff
@@ -28,6 +28,8 @@ You also need a `PSR-7 message implementation`_. If you use Guzzle 6, Guzzle’s
 implementation is already included. If you use another client, install one of
 the implementations. Recommended:
 
+TODO: i think we need the matching adapter for the factory, if guzzle6-adapter is not installed
```

Comment:
This section is about the message implementation: ‘You also need…’. Guzzle 6 comes with guzzlehttp/psr7. What I mean by ‘If you use another client, install one of the implementations.’ is this: if you use another HTTP client instead of Guzzle 6, you need to include a PSR-7 implementation yourself. We recommend to require guzzlehttp/psr7, but you can also use e.g. Diactoros.

However, I agree that we can improve clarity here.